modules/publicize.php

```diff
@@ -46,6 +46,10 @@ function __construct() {
 			if ( in_array( 'publicize', $active ) && !in_array( 'sharedaddy', $active ) )
 				add_action( 'admin_menu', array( &$publicize_ui, 'sharing_menu' ) );
 		}
+
+		// TODO: Not really a block. The underlying logic doesn't care, so we should rename to
+		// `jetpack_register_gutenberg_extension()` (to account for both Gutenblocks and Gutenplugins).
+		jetpack_register_block( 'publicize' );
```

Comment:
I agree! This works, but in a future PR we may want to rename stuff for clarity and accuracy.